### ISSUE TITLE
Subscriber Importer: Add a subscriber-facing unsubscribe/opt-out

### ISSUE THREAD

**Issue by inaikem** [Automattic/wp-calypso#69648]
### What

Add a subscriber-facing unsubscribe/opt-out with various options subscribers can use to indicate their reasons for unsubscribing.

### Why

We can use this to help us identify, monitor and manage spam and TOS violation issues. For example, if X% of a user's subscribers flag TOS/spam unsubscribe reasons, the site/user gets added to a review flow.

As an example of what I think is a great implementation, Mailchimp's version is extremely simple to use, has hardly changed over the past 8 years I've used them (an indicator of something working as intended, perhaps?), and provides overlaps where single terms may be confusing/ambiguous to some audiences.

![NeverSubscribed](https://user-images.githubusercontent.com/66711744/199179268-88b12e37-33d1-4427-9314-ec6faaa08b6f.png)

**Example:** 
- I no longer want... = Typical unsubscribe
- I never signed up.... = Spammer indicator
- The emails are inappropriate = Potential TOS violation indicator
- The emails are spam... = Spammer indicator

### How

We could add this as its own page, a modal in Calypso or (ideally not), at https://subscribe.wordpress.com.

**Comment by vishnugopal** [comment]
Right now the interaction looks like this:
<img width="1075" alt="CleanShot 2022-11-01 at 12 21 20@2x" src="https://user-images.githubusercontent.com/533/199189219-53a81113-f693-445e-9ad7-e32d97339632.png">
<img width="1095" alt="CleanShot 2022-11-01 at 12 23 49@2x" src="https://user-images.githubusercontent.com/533/199189215-08530073-820c-4c7c-82df-4c1e99742bed.png">

**Comment by saygunnyc** [comment]
@vishnugopal I've been reviewing the existing experience here – inDLaEQV8jJ21O4WXawIsJ-fi-2%3A101 How did you manage to open that Delete Your Subscription UI? In my case, as soon as I click unfollow, it just notifies me it's done.

**Comment by vishnugopal** [comment]
![Image from iOS](https://user-images.githubusercontent.com/533/199883217-13ab4475-843a-4f32-9d50-4c8b105b320b.jpg)

I clicked on either one of these links in the email. It's a bit strange that we're landing at different screens :-/

**Comment by vishnugopal** [comment]
![Image from iOS](https://user-images.githubusercontent.com/533/199883217-13ab4475-843a-4f32-9d50-4c8b105b320b.jpg)

I clicked on either one of these links in the email. It's a bit strange that we're landing at different screens :-/

**Comment by vishnugopal** [comment]
Moving to Apex as per this discussion on Slack: p1669134700118009/1669061915.799559-slack-C0442CPG81M

**Comment by saygunnyc** [comment]
Yes, that might need further investigation. I do have the separation for paid subscriptions v.s. free subscriptions can land on this Figma. –Paid subscriptions might require a slightly different flow (directly unsubscribing v.s. going into a flow similarly to WordPress.com plan cancellations) and perhaps be hosted on purchase management.

<img width="2239" alt="FreeSubscription" src="https://user-images.githubusercontent.com/78876323/204393219-9aa24983-a092-4693-894f-b9e7507abe6f.png">
<img width="2239" alt="Paid Subscriptions" src="https://user-images.githubusercontent.com/78876323/204393221-c3a23bed-84ba-476c-b53d-2f9edcb3a4d0.png">
